app-tasks/rf/src/rf/uploads/sentinel2/create_thumbnails.py

```diff
@@ -0,0 +1,27 @@
+"""Creates thumbnails for sentinel 2 imagery
+
+At this point raster foundry reuses the preview
+thumbnail in the tile directory
+"""
+
+from .settings import base_http_path, organization
+from rf.models import Thumbnail
+
+
+def create_thumbnails(tile_path):
+    """Creates thumbnail based on tile_path
+
+    Args:
+        tile_path (str): path to tile directory (e.g. tiles/54/M/XB/2016/9/25/0)
+
+    Returns:
+        List[Thumbnail]
+    """
+    key_path = '{tile_path}/preview.jpg'.format(tile_path=tile_path)
+    thumbnail_url = base_http_path.format(key_path=key_path)
```

Comment:
Do we want to make a HEAD query to verify that the thumbnail actually exists? Many of the older tiles appear to have broken thumbnail links.  
Alternatively, we can handle it by making the image query in javascript, then displaying either the image or a placeholder if the query fails/doesn't have a thumbnail.